arraycontext/impl/pytato/compile.py

```diff
@@ -154,7 +154,34 @@ def id_collector(keys, ary):
     return pmap(arg_id_to_arg), pmap(arg_id_to_descr)
 
 
-def _get_f_placeholder_args(arg, kw, arg_id_to_name):
+def _transform_array(ary, actx):
```

Comment:
Not loving the name. `_to_input_for_compiled`?

Reply:
Yep, that's better. Fixed!